### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by mcapuccini**
## Change content and motivation
Fixing a very minor thing in the contributing document.

### CODE REVIEW

**Comment by mcapuccini** [comment]
@carmat88 please wait for someone review before merging PRs. There is a very minor thing in the document that I spotted.

**Review by carmat88 — APPROVED** [review]

**Pull request merged**
